jewelbots_nRF51822_board_v100/cores/JWB_nRF51822/Timer.h

```diff
@@ -12,7 +12,7 @@ extern "C"{
     Timer();
     ~Timer();
     void pause(uint32_t milliseconds);
-
+    uint32_t millis(void);
```

Comment:
maybe make this a more descriptive name like `millis_since_activated` or `run_time_millis`